ktor-features/ktor-webjars/jvm/src/io/ktor/webjars/Webjars.kt

```diff
@@ -22,6 +22,9 @@ import java.time.*
 @Suppress("KDocMissingDocumentation")
 @KtorExperimentalAPI
 class Webjars(private val configuration: Configuration) {
+    private val locator = WebJarAssetLocator()
+    private val knownWebJars = locator.webJars?.keys?.toSet() ?: emptySet()
+    private val lastModified = ZonedDateTime.now(configuration.zone)
```

Comment:
Is it ok to use `jdk > 6` here?

Reply:
Nobody complained, it actually never worked before. There is also `ZoneId` usage. We could eliminate it separately.